quadra/tasks/classification.py

```diff
@@ -775,26 +769,34 @@ def prepare(self) -> None:
 
         self.config.datamodule.class_to_idx = class_to_idx
 
-        # Setup datamodule
         self.datamodule = self.config.datamodule
+        # prepare_data() must be explicitly called because there is no lightning training
         self.datamodule.prepare_data()
         self.datamodule.setup(stage="test")
 
         self.test_dataloader = self.datamodule.test_dataloader()
+
+        # Configure trainer
+        self.trainer = self.config.trainer
+
+    @property
+    def deployment_model(self):
+        """Deployment model."""
+        return None
+
+    @deployment_model.setter
+    def deployment_model(self, model_path: str):
+        """Set backbone and classifier."""
         try:
             self.backbone = OmegaConf.load(
```

Comment:
I don't get it, why should we be forced to use pytorch here? 
I think we should support any kind of model as per other tasks, if torch is not used we don't generate gradients.

Or am I missing something?

Reply:
See commit "fix: Model import in sklearn evaluations now supports .pt and .pth model files" and tell me what you think